kotsadm/web/src/components/apps/AppGitops.jsx

```diff
@@ -105,33 +102,22 @@ class AppGitops extends Component {
           Utilities.logoutUser();
           return;
         }
-        let msg = `Unexpected status code: ${res.status}`;
+        this.props.refetch();
```

Comment:
moving this here would only refetch if `!res.ok`

Reply:
that is how it was before https://github.com/replicatedhq/kots/blame/5ccc4e16229c9066ed33fb3224697406c14e7398/kotsadm/web/src/components/apps/AppGitops.jsx#L109